lib/main.coffee

```diff
@@ -77,9 +88,10 @@ module.exports =
             '--output-format=text'
           ]
           if @rcFile
-            args.push "--rcfile=#{@rcFile}"
+            rcFile = @rcFile.replace(/%p/g, projDir).replace(/%f/g, path.dirname(file))
```

Comment:
I figured that since we're just searching for a specific string to replace this probably performs better and requires one less character, not that it makes much difference.